src/plugins/common/ui/lib/settings.js

```diff
@@ -104,6 +104,7 @@ define([
 				'toggleMetaView',
 				'toggleFormatlessPaste',
 				'formatNumeratedHeaders',
+				'gcnSpellCheck',
```

Comment:
Our `gcn` plugin isn't part of Aloha-Editor itself, therefore this config entry shouldn't be here.
Please remove this entry here, and refer to the review in the CMS Repo